### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by Unleashedicon**
### my first code review
review my index.html file and styles.css file

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+                    
+                   
+                    <p>Follow Rema</p>
+                    <ul>
+                        <li>TikTok:<a href="#"> https://www.tiktok.com/@badboyrema</a></li>
+                        <li>Instagram:<a href="#"> https://www.instagram.com/heisrema/</a></li>
+                        <li>Facebook:<a href="#"> https://www.facebook.com/heisrema</a></li>
+                        <li>Twitter:<a href="#"> https://twitter.com/heisrema</a></li>
+                        <li>SoundCloud:<a href="#"> https://soundcloud.com/heisrema</a></li>
+                    </ul>
```
Comment (Unleashedicon) [inline index.html:79]: I would suggest you enclose the following in a nav tag. It is used to create a semantic grouping of navigation-related links, which can improve accessibility and make it easier for users to understand the structure and organization of a website.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+        <div class="container">
+            <ul>
+              <li><a href="#">About</a></li>
+              <li><a href="#">Press</a></li>
+              <li><a href="#">Copyright</a></li>
+              <li><a href="#">Contact</a></li>
+              <li><a href="#">Creators</a></li>
+              <li><a href="#">Advertising</a></li>
+              <li><a href="#">Developers</a></li>
+            </ul>
```
Comment (Unleashedicon) [inline index.html:145]: i would suggest also closing this with a nav tag

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+    position: absolute;
+    right: 100px;
+    top: 10px;
+  }
+  
+  .Account {
+    position: absolute;
+    right: 100px;
+    top: 10px;
+  }
```
Comment (Unleashedicon) [inline styles.css:91]: can you please finish styling your code in order to provide a more appealing website

File: styles.css [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+    width: 100%;
+    height: 70px;
+    background-color: #fff;
+    box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
+    position: fixed;
+    top: 0;
+    left: 0;
+    z-index: 1;
+    display: flex;
+    align-items: center;
```
Comment (Unleashedicon) [inline styles.css:28]: check on indentation to avoid linter errors

**Review by Unleashedicon — COMMENTED** [review]
### changes required ♻️ 
Great commits but you can still try to improve your project more 😃 
check the comments below and make the required changes

you should also check on linters as you may experience linter errors

Make the changes and then submit it for a re-review 👍

**Review by Olamarx — APPROVED** [review]
Hi @Unleashedicon,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Polite comments ✔️ 
- Reasons for comments ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**